### PR TITLE
 adding test for skipfeevalidation

### PR DESCRIPTION
**Pull request by kameshraj23**
Test added for skipfee vlaidation 

Screenshot 2024-01-26 at 09 48 08

### CODE REVIEW

File: test/endpoint_test.go [hunk 1]
```diff
@@ -14,8 +13,12 @@ import (
 	"testing"
 	"time"
 
+	// "time"
```
Comment (boecklim) [inline test/endpoint_test.go:16]: These comments are not needed

File: test/endpoint_test.go [hunk 2]
```diff
@@ -14,8 +13,12 @@ import (
...
+
 	"github.com/bitcoin-sv/arc/api"
 	"github.com/bitcoin-sv/arc/api/handler"
+
+	// "github.com/bitcoin-sv/arc/api/handler"
```
Comment (boecklim) [inline test/endpoint_test.go:21]: These comments are not needed

File: test/endpoint_test.go [hunk 3]
```diff
@@ -110,65 +120,65 @@ func createTx(privateKey string, address string, utxo NodeUnspentUtxo) (*bt.Tx,
...
 
-func TestBatchChainedTxs(t *testing.T) {
-	tt := []struct {
-		name string
-	}{
-		{
-			name: "submit batch of chained transactions - ext format",
-		},
-	}
+// func TestBatchChainedTxs(t *testing.T) {
```
Comment (boecklim) [inline test/endpoint_test.go:123]: Please uncomment this test

File: test/endpoint_test.go [hunk 4]
```diff
@@ -409,6 +419,75 @@ func TestPostCallbackToken(t *testing.T) {
...
+			fmt.Println("Transaction with Zero fee:", tx)
+
+			url := "http://arc:9090/"
+
+			arcClient, err := api.NewClientWithResponses(url)
+			require.NoError(t, err)
+
+			ctx := context.Background()
+
+			// hostname, err := os.Hostname()
```
Comment (boecklim) [inline test/endpoint_test.go:464]: Not useful comments. Please remove

Comment (kameshraj23) [reply test/endpoint_test.go]: Addressed

File: test/endpoint_test.go [hunk 5]
```diff
@@ -62,7 +63,7 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func createTx(privateKey string, address string, utxo NodeUnspentUtxo) (*bt.Tx, error) {
+func createTx(privateKey string, address string, utxo NodeUnspentUtxo, fee ...uint64) (*bt.Tx, error) {
```
Comment (boecklim) [inline test/endpoint_test.go:66]: For one tx you can have only one fee.
I suggest to change this to `createTx(privateKey string, address string, utxo NodeUnspentUtxo, fee uint64)`

**Pull request merged**